legend-pure-m2-store-relational/src/main/java/org/finos/legend/pure/m2/relational/serialization/grammar/v1/processor/DatabaseProcessor.java

```diff
@@ -194,59 +194,59 @@ static void processTable(CoreInstance table, Matcher matcher, ProcessorState pro
-            ((Table)table)._primaryKey(primaryKey);
+            ((Table) table)._primaryKey(primaryKey);
         }
 
         if (table instanceof Relation)
         {
-            Relation relation = (Relation)table;
-            relation._setColumnsCoreInstance(relation._columns().collect(relationalOperationElement -> (Column)relationalOperationElement));
+            Relation relation = (Relation) table;
+            relation._setColumnsCoreInstance(relation._columns().collect(relationalOperationElement -> (Column) relationalOperationElement));
         }
         processTableMilestoning(table, columnsByName, matcher, processorState, processorSupport);
     }
 
     public static MapIterable<String, Column> processColumnsForTableOrView(CoreInstance tableOrView)
     {
-        ListIterable<? extends RelationalOperationElement> columns = FastList.newList();
         if (tableOrView instanceof Relation)
         {
-            columns = ((Relation)tableOrView)._columns().toList();
-        }
-        MutableMap<String, Column> columnsByName = UnifiedMap.newMap(columns.size());
-        for (RelationalOperationElement column : columns)
-        {
-            if (!(column instanceof Column))
-            {
-                throw new PureCompilationException(column.getSourceInformation(), "Expected an instance of " + M2RelationalPaths.Column + ", found " + PackageableElement.getUserPathForPackageableElement(column.getClassifier()));
-            }
-            String columnName = ((Column)column)._name();
-            RelationalOperationElement old = columnsByName.put(columnName, (Column)column);
-            if (old != null)
+            ListIterable<? extends RelationalOperationElement> columns = ((Relation) tableOrView)._columns().toList();
+            MutableMap<String, Column> columnsByName = UnifiedMap.newMap(columns.size());
+            for (RelationalOperationElement column : columns)
             {
-                throw new PureCompilationException(column.getSourceInformation(), "Multiple columns named '" + columnName + "' found in " + tableOrView.getClassifier().getName().toLowerCase() + " " + ((NamedRelation)tableOrView)._name());
+                if (!(column instanceof Column))
+                {
+                    throw new PureCompilationException(column.getSourceInformation(), "Expected an instance of " + M2RelationalPaths.Column + ", found " + PackageableElement.getUserPathForPackageableElement(column.getClassifier()));
+                }
+                String columnName = ((Column) column)._name();
+                RelationalOperationElement old = columnsByName.put(columnName, (Column) column);
+                if (old != null)
+                {
+                    throw new PureCompilationException(column.getSourceInformation(), "Multiple columns named '" + columnName + "' found in " + tableOrView.getClassifier().getName().toLowerCase() + " " + (tableOrView.getValueForMetaPropertyToOne(M3Properties.name).getName()));
```

Comment:
If tableOrView is not a NamedRelation, then there will be no "name" property and this will throw a NullPointerException. We need better logic here to avoid throwing an exception while generating an exception message.